### PR TITLE
Manifest generation

### PR DESCRIPTION
**Pull request by stepango**
Generate default manifest files using packageName.

My initial attempt was to make a cachable Gradle task for it. But Manifest file has to exist in the configuration stage.

### CODE REVIEW

File: plugins/android/src/main/java/tools/forma/android/feature/AndroidLibrary.kt [hunk 1]
```diff
@@ -24,9 +26,33 @@ fun androidLibraryFeatureDefinition(
...
+                 * Some naive caching here, file name equals package name
+                 * We create new file only if package is changed or on first build
+                 */
+                with(File(project.buildDir, "tmp/manifest/${feature.packageName}.xml")) {
+                    if (!exists()) {
+                        parentFile.mkdirs()
+                        createNewFile()
+                        writeText(
+                            """<?xml version="1.0" encoding="utf-8"?>
+                           <manifest package="${feature.packageName}"/>
```
Comment (KarenkovID) [inline plugins/android/src/main/java/tools/forma/android/feature/AndroidLibrary.kt:43]: What about support theme for module? We have to add themes for correct layout/resources preview in android studio 
```xml
<application android:theme="@style/AppTheme.Mobile"/>
```

Comment (stepango) [reply plugins/android/src/main/java/tools/forma/android/feature/AndroidLibrary.kt:43]: You can choose the theme on the preview screen, no?

Comment (stepango) [reply plugins/android/src/main/java/tools/forma/android/feature/AndroidLibrary.kt:43]: Screen Shot 2021-01-31 at 10 41 47 PM

File: plugins/android/src/main/java/tools/forma/android/feature/AndroidLibrary.kt [hunk 2]
```diff
@@ -2,20 +2,22 @@ package tools.forma.android.feature
...
 import tools.forma.validation.Validator
 import tools.forma.validation.validator
 
-data class AndroidLibraryFeatureConfiguration(
+class AndroidLibraryFeatureConfiguration(
     val packageName: String,
     val buildConfiguration: BuildConfiguration = BuildConfiguration(),
     val testInstrumentationRunnerClass: String = androidJunitRunner,
     val consumerMinificationFiles: Set<String> = emptySet(),
     val manifestPlaceholders: Map<String, Any> = emptyMap(),
```
Comment (michaem) [inline plugins/android/src/main/java/tools/forma/android/feature/AndroidLibrary.kt:17]: Shouldn't we relate manifestPlaceholders logic with generating tmp manifest?

**Review by michaem — APPROVED** [review]

**Pull request merged**